### PR TITLE
Comparison table updates

### PR DESCRIPTION
**Pull request by davidrapson**
This PR is a re-cut of https://github.com/guardian/membership-frontend/pull/700 with the new benefits removed and the comparison table structure refactored to avoid this bug: https://github.com/guardian/membership-frontend/issues/717. There's a few magic numbers in the CSS but it's largely no more complex than the previous approach.

screen shot 2015-09-03 at 14 52 40
screen shot 2015-09-03 at 14 52 30
screen shot 2015-09-03 at 14 52 48

@afiore

### CODE REVIEW

File: frontend/app/model/Benefits.scala [hunk 1]
```diff
@@ -83,43 +92,36 @@ object Benefits {
...
-    """Show deep support for keeping the Guardian free, open and independent.
-    | Get invited to a small number of exclusive, behind-the-scenes functions""".stripMargin
+
+  val comparisonHiglightsList = benefitsFilter(
+    "priority_booking",
+    "discount",
+    "unique_experiences"
   )
 
   def details(tier: Tier) = tier match {
```
Comment (tudorraul) [inline frontend/app/model/Benefits.scala]: This looks to be related only to `views`, so it should not be a present in `model`

File: frontend/assets/stylesheets/components/_icons.scss [hunk 2]
```diff
@@ -61,6 +61,12 @@
     width: 36px;
     height: 36px;
 }
+.icon-inline--badge {
+    width: 50px;
```
Comment (tudorraul) [inline frontend/assets/stylesheets/components/_icons.scss]: are `px`s automatically converted to `rem`s?

**Comment by tudorraul** [comment]
:+1: bar the comments

File: frontend/app/model/Benefits.scala (hunk 1 above)
Comment (davidrapson) [inline frontend/app/model/Benefits.scala]: Given it's so tightly related to this model, is it not preferable to live here?

File: frontend/assets/stylesheets/components/_icons.scss (hunk 2 above)
Comment (davidrapson) [inline frontend/assets/stylesheets/components/_icons.scss]: Yeah, we have a task for it now https://github.com/guardian/membership-frontend/blob/master/frontend/Gruntfile.js#L71

**Pull request merged**